app/src/features/apiClient/screens/apiClient/components/sidebar/apiClientSidebarTabs/historyList/HistoryList.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useCallback } from "react";
+import placeholderImage from "../../../../../../../../assets/images/illustrations/empty-sheets-dark.svg";
+import { RQAPI } from "features/apiClient/types";
+import { ClearOutlined } from "@ant-design/icons";
+import { RQButton } from "lib/design-system-v2/components";
+import { Timeline, Typography } from "antd";
+import { REQUEST_METHOD_COLORS } from "../../../../../../../../constants";
+import { trackRequestSelectedFromHistory } from "modules/analytics/events/features/apiClient";
+import { trackRQDesktopLastActivity, trackRQLastActivity } from "utils/AnalyticsUtils";
+import { API_CLIENT } from "modules/analytics/events/features/constants";
+
+interface Props {
+  history: RQAPI.Entry[];
+  onSelectionFromHistory: (index: number) => void;
+  clearHistory: () => void;
```

Comment:
Lets keep the naming convention same for functions passed as props.
Rename this to `onClearHistory`.